### PR TITLE
removed unused resources (classes, XML, images) (#45)

### PR DESCRIPTION
**Pull request by timecomplexity**
#45

### CODE REVIEW

File: app/src/main/java/pbell/offline/ole/org/pbell/FullscreenActivity.java [hunk 1]
```diff
@@ -1768,7 +1769,8 @@ public boolean updateActivityOpenedResources(String resource_name, String resour
             manager = new Manager(androidContext, Manager.DEFAULT_OPTIONS);
             activityLog = manager.getDatabase("activitylog");
             WifiManager wm = (WifiManager) getSystemService(Context.WIFI_SERVICE);
-            String m_WLANMAC = wm.getConnectionInfo().getMacAddress();
+            // String m_WLANMAC = wm.getConnectionInfo().getMacAddress();
+            String m_WLANMAC = "mymac";
```
Comment (kylemathias) [inline app/src/main/java/pbell/offline/ole/org/pbell/FullscreenActivity.java]: I dont know what to do about that. gotta remove that change

File: app/src/main/java/pbell/offline/ole/org/pbell/FullscreenLogin.java [hunk 2]
```diff
@@ -611,7 +612,8 @@ public boolean updateActivityLog() {
             manager = new Manager(androidContext, Manager.DEFAULT_OPTIONS);
             activityLog = manager.getDatabase("activitylog");
             WifiManager wm = (WifiManager) getSystemService(Context.WIFI_SERVICE);
-            String m_WLANMAC = wm.getConnectionInfo().getMacAddress();
+            // String m_WLANMAC = wm.getConnectionInfo().getMacAddress();
+            String m_WLANMAC = "mymac";
```
Comment (kylemathias) [inline app/src/main/java/pbell/offline/ole/org/pbell/FullscreenLogin.java]: same here.

**Review by kylemathias — COMMENTED** [review]
gotta remember to remove those lines.

**Comment by timecomplexity** [comment]
Admittedly a little frustrating, but easy to fix. Maybe we can find a way around this in the future?

**Comment by kylemathias** [comment]
yea, im thinking about what to do. i know there is a git ignore function, but i dont really know much about it.

**Comment by kylemathias** [comment]
and on that note, how do i even go about testing this branch? lol

**Comment by timecomplexity** [comment]
I'm not really sure how you would test this one since I just deleted a bunch of stuff, @kylemathias .
I suppose you could just test it to see if it compiles and runs? haha.

**Review by leomaxi — APPROVED** [review]

**Pull request merged**